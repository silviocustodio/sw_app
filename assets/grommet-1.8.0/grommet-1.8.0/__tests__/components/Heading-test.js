// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP

import React from 'react';
import renderer from 'react-test-renderer';

import Heading from '../../src/js/components/Heading';

describe('Heading', () => {
  it('has correct default options', () => {
    const component = renderer.create(
      <Heading>Testing</Heading>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('has correct strong=true and align="center" rendering', () => {
    const component = renderer.create(
      <Heading strong={true} align='center'>Testing</Heading>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('has correct size="large" and uppercase=true rendering', () => {
    const component = renderer.create(
      <Heading size='large' uppercase={true}>Testing</Heading>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('has correct margin="small" and className rendering', () => {
    const component = renderer.create(
      <Heading margin='small' className='testing'>Testing</Heading>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
