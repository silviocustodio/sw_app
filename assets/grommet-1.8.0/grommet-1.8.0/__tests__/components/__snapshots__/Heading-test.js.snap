// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading has correct default options 1`] = `
<h1
  className="grommetux-heading"
>
  Testing
</h1>
`;

exports[`Heading has correct margin="small" and className rendering 1`] = `
<h1
  className="grommetux-heading grommetux-heading--margin-small testing"
>
  Testing
</h1>
`;

exports[`Heading has correct size="large" and uppercase=true rendering 1`] = `
<h1
  className="grommetux-heading grommetux-heading--large grommetux-heading--uppercase"
>
  Testing
</h1>
`;

exports[`Heading has correct strong=true and align="center" rendering 1`] = `
<h1
  className="grommetux-heading grommetux-heading--strong grommetux-heading--align-center"
>
  Testing
</h1>
`;
